coalahtml/tree.py

```diff
@@ -0,0 +1,81 @@
+from collections import OrderedDict
+from queue import Queue
+import os
+
+
+class Node:
+    """
+    Node class that defines an individual node.
```

Comment:
It's clear without this that this is node class to define individual node. You can go without docstring, but if you want to keep it, make it useful.